用户id  : 1
16/dst/ser006img00715.png
16/dst/ser006img00716.png
16/dst/ser006img00717.png
[pid: 334073|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:12:59 2024] GET /api/medical/get_ct?patient_id=1&case_id=16 => generated 256 bytes in 142 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
用户id  : 1
16/dst/ser006img00715.png
16/dst/ser006img00716.png
16/dst/ser006img00717.png
[pid: 334073|app: 0|req: 2/2] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:15:35 2024] GET /api/medical/get_ct?patient_id=1&case_id=16 => generated 256 bytes in 109 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 1)
...gracefully killing workers...
Gracefully killing worker 1 (pid: 334069)...
Gracefully killing worker 2 (pid: 334070)...
Gracefully killing worker 4 (pid: 334073)...
Gracefully killing worker 3 (pid: 334072)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:16:03 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x55d23e099f70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55d23e099f70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 335127, cores: 2)
spawned uWSGI worker 2 (pid: 335128, cores: 2)
spawned uWSGI worker 3 (pid: 335130, cores: 2)
spawned uWSGI worker 4 (pid: 335131, cores: 2)
用户id  : 1
16/dst/ser006img00715.png
Internal Server Error: /api/medical/get_ct
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/django/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ubuntu/anaconda3/envs/django/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/chuangxin/vasorum-detacted-backend/./user/authentication.py", line 27, in valid_per
    return func(request, *args, **kwargs)
  File "/home/ubuntu/chuangxin/vasorum-detacted-backend/./medical/views.py", line 250, in get_ct
    print("哈哈"+len(segs))
TypeError: can only concatenate str (not "int") to str
[pid: 335130|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:16:07 2024] GET /api/medical/get_ct?patient_id=1&case_id=16 => generated 61929 bytes in 251 msecs (HTTP/1.1 500) 7 headers in 242 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 1 (pid: 335127)...
Gracefully killing worker 2 (pid: 335128)...
Gracefully killing worker 4 (pid: 335131)...
Gracefully killing worker 3 (pid: 335130)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:16:28 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x55bd71b0ef70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55bd71b0ef70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 335309, cores: 2)
spawned uWSGI worker 2 (pid: 335310, cores: 2)
spawned uWSGI worker 3 (pid: 335312, cores: 2)
spawned uWSGI worker 4 (pid: 335313, cores: 2)
用户id  : 1
16/dst/ser006img00715.png
哈哈 0
16/dst/ser006img00716.png
哈哈 0
16/dst/ser006img00717.png
哈哈 0
[pid: 335310|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:16:30 2024] GET /api/medical/get_ct?patient_id=1&case_id=16 => generated 256 bytes in 145 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 1 (pid: 335309)...
Gracefully killing worker 2 (pid: 335310)...
Gracefully killing worker 3 (pid: 335312)...
Gracefully killing worker 4 (pid: 335313)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:17:32 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x558eb1fbbf70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x558eb1fbbf70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 335674, cores: 2)
spawned uWSGI worker 2 (pid: 335675, cores: 2)
spawned uWSGI worker 3 (pid: 335677, cores: 2)
spawned uWSGI worker 4 (pid: 335678, cores: 2)
用户id  : 1
16/dst/ser006img00715.png
193
哈哈 0
16/dst/ser006img00716.png
194
哈哈 0
16/dst/ser006img00717.png
195
哈哈 0
[pid: 335677|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:17:34 2024] GET /api/medical/get_ct?patient_id=1&case_id=16 => generated 256 bytes in 143 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
用户id  : 1
[pid: 335678|app: 0|req: 1/2] 219.239.227.174 () {42 vars in 837 bytes} [Thu Apr 11 19:21:38 2024] POST /api/medical/upload => generated 407 bytes in 211 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
用户id  : 1
17
ser006img00715.jpg
Internal Server Error: /api/medical/upload_seg
Traceback (most recent call last):
  File "/home/ubuntu/anaconda3/envs/django/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ubuntu/anaconda3/envs/django/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ubuntu/chuangxin/vasorum-detacted-backend/./user/authentication.py", line 27, in valid_per
    return func(request, *args, **kwargs)
  File "/home/ubuntu/chuangxin/vasorum-detacted-backend/./medical/views.py", line 182, in upload_seg
    photo = Photo.objects.get(ct=ct, img__contains=photo_name,path="dst")
  File "/home/ubuntu/anaconda3/envs/django/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/ubuntu/anaconda3/envs/django/lib/python3.10/site-packages/django/db/models/query.py", line 649, in get
    raise self.model.DoesNotExist(
medical.models.Photo.DoesNotExist: Photo matching query does not exist.
[pid: 335675|app: 0|req: 1/3] 219.239.227.174 () {42 vars in 841 bytes} [Thu Apr 11 19:22:10 2024] POST /api/medical/upload_seg => generated 71776 bytes in 174 msecs (HTTP/1.1 500) 7 headers in 242 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 1 (pid: 335674)...
Gracefully killing worker 2 (pid: 335675)...
Gracefully killing worker 4 (pid: 335678)...
Gracefully killing worker 3 (pid: 335677)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:28:40 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x5587ee1a5f70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x5587ee1a5f70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 339303, cores: 2)
spawned uWSGI worker 2 (pid: 339304, cores: 2)
spawned uWSGI worker 3 (pid: 339306, cores: 2)
spawned uWSGI worker 4 (pid: 339307, cores: 2)
用户id  : 1
17
ser006img00715.jpg
17
ser006img00716.jpg
17
ser006img00717.jpg
17
ser006img00715.jpg
17
ser006img00716.jpg
17
ser006img00717.jpg
[pid: 339307|app: 0|req: 1/1] 219.239.227.174 () {42 vars in 841 bytes} [Thu Apr 11 19:28:42 2024] POST /api/medical/upload_seg => generated 65 bytes in 277 msecs (HTTP/1.1 200) 7 headers in 212 bytes (1 switches on core 0)
用户id  : 1
17/dst/ser006img00715.png
199
哈哈 2
17/dst/ser006img00716.png
200
哈哈 2
17/dst/ser006img00717.png
201
哈哈 2
[pid: 339306|app: 0|req: 1/2] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:28:59 2024] GET /api/medical/get_ct?patient_id=1&case_id=17 => generated 256 bytes in 155 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 1 (pid: 339303)...
Gracefully killing worker 2 (pid: 339304)...
Gracefully killing worker 3 (pid: 339306)...
Gracefully killing worker 4 (pid: 339307)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:30:54 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x5561efba8f70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5561efba8f70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 340158, cores: 2)
spawned uWSGI worker 2 (pid: 340159, cores: 2)
spawned uWSGI worker 3 (pid: 340161, cores: 2)
spawned uWSGI worker 4 (pid: 340162, cores: 2)
用户id  : 1
17/dst/ser006img00715.png
199
哈哈 2
http://101.42.32.89/media/17/dst/src17/dst/src/1-001-D-1208051-ser006img00715.jpg
http://101.42.32.89/media/17/dst/src17/dst/src/2-001-D-1208051-ser006img00715.jpg
17/dst/ser006img00716.png
200
哈哈 2
http://101.42.32.89/media/17/dst/src17/dst/src/1-001-D-1208051-ser006img00716.jpg
http://101.42.32.89/media/17/dst/src17/dst/src/2-001-D-1208051-ser006img00716.jpg
17/dst/ser006img00717.png
201
哈哈 2
http://101.42.32.89/media/17/dst/src17/dst/src/1-001-D-1208051-ser006img00717.jpg
http://101.42.32.89/media/17/dst/src17/dst/src/2-001-D-1208051-ser006img00717.jpg
[pid: 340159|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:31:02 2024] GET /api/medical/get_ct?patient_id=1&case_id=17 => generated 256 bytes in 147 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 2 (pid: 340159)...
Gracefully killing worker 1 (pid: 340158)...
Gracefully killing worker 3 (pid: 340161)...
Gracefully killing worker 4 (pid: 340162)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:32:19 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x5620bc9abf70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5620bc9abf70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 340704, cores: 2)
spawned uWSGI worker 2 (pid: 340705, cores: 2)
spawned uWSGI worker 3 (pid: 340706, cores: 2)
spawned uWSGI worker 4 (pid: 340708, cores: 2)
...gracefully killing workers...
Gracefully killing worker 4 (pid: 340708)...
Gracefully killing worker 1 (pid: 340704)...
Gracefully killing worker 2 (pid: 340705)...
Gracefully killing worker 3 (pid: 340706)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:32:36 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x559907e9df70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x559907e9df70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 340840, cores: 2)
spawned uWSGI worker 2 (pid: 340841, cores: 2)
spawned uWSGI worker 3 (pid: 340843, cores: 2)
spawned uWSGI worker 4 (pid: 340844, cores: 2)
用户id  : 1
17/dst/ser006img00715.png
哈哈 2
17/dst/ser006img00716.png
哈哈 2
17/dst/ser006img00717.png
哈哈 2
[pid: 340844|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:32:39 2024] GET /api/medical/get_ct?patient_id=1&case_id=17 => generated 1007 bytes in 146 msecs (HTTP/1.1 200) 7 headers in 214 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 4 (pid: 340844)...
Gracefully killing worker 1 (pid: 340840)...
Gracefully killing worker 2 (pid: 340841)...
Gracefully killing worker 3 (pid: 340843)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:36:02 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x5631f4833f70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x5631f4833f70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 342194, cores: 2)
spawned uWSGI worker 2 (pid: 342195, cores: 2)
spawned uWSGI worker 3 (pid: 342197, cores: 2)
spawned uWSGI worker 4 (pid: 342198, cores: 2)
Not Found: /api/api/medical/annotate
[pid: 342197|app: 0|req: 1/1] 219.239.227.174 () {40 vars in 578 bytes} [Thu Apr 11 19:36:04 2024] POST /api/api/medical/annotate => generated 2496 bytes in 61 msecs (HTTP/1.1 404) 7 headers in 229 bytes (1 switches on core 0)
用户id  : 1
17/dst/ser006img00715.png
哈哈 2
17/dst/ser006img00716.png
哈哈 2
17/dst/ser006img00717.png
哈哈 2
[pid: 342198|app: 0|req: 1/2] 219.239.227.174 () {38 vars in 667 bytes} [Thu Apr 11 19:36:19 2024] GET /api/medical/get_ct?patient_id=1&case_id=17 => generated 1026 bytes in 145 msecs (HTTP/1.1 200) 7 headers in 214 bytes (1 switches on core 0)
用户id  : 5
[pid: 342197|app: 0|req: 2/3] 219.239.227.192 () {44 vars in 907 bytes} [Thu Apr 11 19:39:06 2024] GET /api/medical/get_case?patient_id=1 => generated 625 bytes in 140 msecs (HTTP/1.1 200) 9 headers in 305 bytes (1 switches on core 1)
用户id  : 5
1/dst/2-德育.png
哈哈 0
1/dst/6-志愿服务_91h.png
哈哈 0
[pid: 342195|app: 0|req: 1/4] 219.239.227.192 () {44 vars in 923 bytes} [Thu Apr 11 19:39:06 2024] GET /api/medical/get_ct?patient_id=1&case_id=1 => generated 370 bytes in 175 msecs (HTTP/1.1 200) 9 headers in 305 bytes (1 switches on core 0)
...gracefully killing workers...
Gracefully killing worker 2 (pid: 342195)...
Gracefully killing worker 1 (pid: 342194)...
Gracefully killing worker 4 (pid: 342198)...
Gracefully killing worker 3 (pid: 342197)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
binary reloading uWSGI...
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
closing all non-uwsgi socket fds > 2 (max_fd = 1048576)...
found fd 6 mapped to socket 0 (127.0.0.1:8000)
running /usr/local/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.23 (64bit) on [Thu Apr 11 19:40:36 2024] ***
compiled with version: 11.2.0 on 15 January 2024 06:07:54
os: Linux-5.15.0-91-generic #101-Ubuntu SMP Tue Nov 14 13:30:08 UTC 2023
nodename: VM-16-17-ubuntu
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /home/ubuntu/chuangxin/vasorum-detacted-backend
detected binary path: /usr/local/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
chdir() to /home/ubuntu/chuangxin/vasorum-detacted-backend
your processes number limit is 7401
your memory page size is 4096 bytes
detected max file descriptor number: 1048576
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address 127.0.0.1:8000 fd 6
Python version: 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Set PythonHome to /home/ubuntu/anaconda3/envs/django
Python main interpreter initialized at 0x5638fb3f9f70
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
added /home/ubuntu/anaconda3/envs/django/lib/python3.10/ to pythonpath.
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x5638fb3f9f70 pid: 2250 (default app)
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 2250)
spawned uWSGI worker 1 (pid: 344098, cores: 2)
spawned uWSGI worker 2 (pid: 344099, cores: 2)
spawned uWSGI worker 3 (pid: 344101, cores: 2)
spawned uWSGI worker 4 (pid: 344102, cores: 2)
用户id  : 1
[pid: 344099|app: 0|req: 1/1] 219.239.227.174 () {38 vars in 638 bytes} [Thu Apr 11 19:40:58 2024] GET /api/medical/get_all_patient => generated 165 bytes in 128 msecs (HTTP/1.1 200) 7 headers in 213 bytes (1 switches on core 0)
[pid: 344102|app: 0|req: 1/2] 180.101.245.248 () {42 vars in 845 bytes} [Thu Apr 11 19:43:19 2024] GET /api/medical/get_case?patient_id=1 => generated 43 bytes in 32 msecs (HTTP/1.1 200) 7 headers in 212 bytes (1 switches on core 0)
